### PR TITLE
ac-3: changed Migrations

### PR DESCRIPTION
**Pull request by tech-akash**
## Issue that this pull request solves

Closes: # (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [ ] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [ ] I have performed a self-review of my own code
-   [ ] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by PranshuNayak** [comment]
@tech-akash give the reference PR's where the changes visible in the migrations file of this PR are done.

**Comment by tech-akash** [comment]
> @tech-akash give the reference PR's where the changes visible in the migrations file of this PR are done.

#1075 #1156

**Comment by PranshuNayak** [comment]
@tech-akash the migrations files are ok. The PR containing the corresponding code changes in models.py i.e. #1156 needs to be merged first, but that PR should not contain any migration file, it should only contain the code changes. After that PR is merged without migrations file this PR can be merged. So do the necessary changes in that #1156 first.

**Review by PranshuNayak — APPROVED** [review]

**Review by samay-rgb — APPROVED** [review]

File: FusionIIIT/applications/online_cms/models.py [hunk 1]
```diff
@@ -186,16 +186,39 @@ class CourseAssignment(models.Model):
...
+#     upload_url = models.TextField(max_length=200)
+#     score = models.IntegerField(null=True)        #score is submitted by faculty 
+#     feedback = models.CharField(max_length=100, null=True)  #feedback by the faculty for the solution of the assignment submitted
+#     assign_name = models.CharField(max_length=100) 
+
+#     def __str__(self):
+#         return '{} - {} - {} - {} - {}'.format(
+#                 self.pk, self.student_id,
+#                 self.assignment_id, self.score,
+#                 self.feedback)
```
Comment (akshatnema) [inline FusionIIIT/applications/online_cms/models.py]: Remove this comment from the PR

File: FusionIIIT/applications/online_cms/models.py [hunk 2]
```diff
@@ -186,16 +186,39 @@ class CourseAssignment(models.Model):
...
+def assignment_submit_name(instance, filename):
+    name, ext = filename.split('.')
+    # obj=Curriculum.objects.get(course_id=instance.course_id)
```
Comment (akshatnema) [inline FusionIIIT/applications/online_cms/models.py]: Same here

File: FusionIIIT/applications/online_cms/views.py [hunk 3]
```diff
@@ -370,28 +373,24 @@ def upload_assignment(request, course_code):
...
+        StudentAssignment1.objects.create(
+            student_id=student,
+            doc=doc,
+            assignment_id=assign,
+            course_code=course_code,
+            assign_name=name+file_extenstion
+            # upload_time=datetime.datetime.now(),
+            # description=description,
+            # document_name=name,
+            # doc=doc
```
Comment (akshatnema) [inline FusionIIIT/applications/online_cms/views.py]:
```suggestion
```
Remove this comment

**Review by akshatnema — CHANGES_REQUESTED** [review]

File: FusionIIIT/applications/online_cms/models.py [hunk 4]
```diff
@@ -29,7 +29,7 @@ class CourseDocuments(models.Model):
     upload_time = models.DateTimeField(auto_now=True)
     description = models.CharField(max_length=100)
     document_name = models.CharField(max_length=40)
-    document_url = models.CharField(max_length=500, null=True,blank=True)
+    document_url = models.CharField(max_length=100, null=True,blank=True)
     # media = models.FileField(upload_to=content_file_name, null=True, blank=True)
```
Comment (akshatnema) [inline FusionIIIT/applications/online_cms/models.py]:
```suggestion
```

File: FusionIIIT/applications/online_cms/models.py [hunk 5]
```diff
@@ -186,16 +186,25 @@ class CourseAssignment(models.Model):
     def __str__(self):
         return '{} - {} - {}'.format(self.pk, self.course_id, self.assignment_name)
 
-#details of the solution uploaded by the student
-class StudentAssignment(models.Model):
+
+def assignment_submit_name(instance, filename):
+    name, ext = filename.split('.')
+    # obj=Curriculum.objects.get(course_id=instance.course_id)
```
Comment (akshatnema) [inline FusionIIIT/applications/online_cms/models.py]:
```suggestion
```

File: FusionIIIT/applications/online_cms/views.py [hunk 6]
```diff
@@ -174,12 +177,12 @@ def course(request, course_code):
         #         videos.append(video_data)
             # print(videos)
```
Comment (akshatnema) [inline FusionIIIT/applications/online_cms/views.py]:
```suggestion
```

File: FusionIIIT/applications/online_cms/views.py [hunk 7]
```diff
@@ -309,16 +312,17 @@ def course(request, course_code):
...
         marks = []
         quizs = []
         assignment = Assignment.objects.select_related().filter(course_id=course)
         assignment1 = CourseAssignment.objects.select_related().filter(course_id=course)
         student_assignment = []
-        for assi in assignment:
-            sa = StudentAssignment.objects.select_related().filter(assignment_id=assi)
+        for assi in assignment1:
+            sa = StudentAssignment1.objects.select_related().filter(assignment_id=assi)
+            # print("hii",sa)
```
Comment (akshatnema) [inline FusionIIIT/applications/online_cms/views.py]:
```suggestion
```

**Review by akshatnema — CHANGES_REQUESTED** [review]

**Review by akshatnema — APPROVED** [review]

**Pull request merged**